src/assets/i18n/en_US.json

```diff
@@ -794,6 +794,16 @@
   "form.password.error.required": "Please enter a password.",
   "form.phone.error.invalid": "Please enter a valid phone number.",
   "form.phone.error.required": "Please enter your phone number.",
+  "gdpr.request.personal.data.confirmation.details": "We will contact you if there are any questions or concerns with your request. You will receive a confirmation e-mail from inTronics with more details about your request once it has been reviewed and processed.",
```

Comment:
avoid to hard code inTRONICS here - take a more general term so other shops don*t have to adapt it like our webmaster or shop